packages/compiler/lib/lib.cadl

```diff
@@ -1,8 +1,31 @@
 namespace Cadl;
 
+@intrinsic("any")
+model any {}
+
+@intrinsic("object")
+model object {}
```

Comment:
Thinking... is this correct as-is (without even the intrinsic) since it's a model without any properties?

Reply:
hhm, right that might be the case, I'll see if that just works.

Reply:
Seems like `object` was added after and a bit confused by the reality of the difference
- `object` says its `not a primitive`
- `{}` is an empty object but there is also an eslint rule by default that suggest not using it
![image](https://user-images.githubusercontent.com/1031227/174349481-525cd610-30e8-4fe8-a445-b3cecf6df729.png)